content/api/events.apib

```diff
@@ -126,6 +126,8 @@ Returns a list of message events that matched the filtered search.
         + Default: `1000`
     + delimiter (string, optional) ... Specifies the delimiter for query parameter lists.
         + Default: `,`
+    + event_ids (string, optional) - Comma delimited list of events, by ID, to search.
+        + Default: `all events`
```

Comment:
I don't think a default value is needed here since it's not technically defaulted - it's just everything. (I'm not sure why `events` has a default though. oh well.)

Reply:
ok, removed